pkg/services/ngalert/api/tooling/definitions/provisioning_mute_timings.go

```diff
@@ -69,8 +69,8 @@ type MuteTimingPayload struct {
 
 // swagger:model
 type MuteTimeInterval struct {
-	config.MuteTimeInterval
-	Provenance models.Provenance `json:"provenance,omitempty"`
+	config.MuteTimeInterval `json:",inline" yaml:",inline"`
```

Comment:
I see how this fixes yaml, but does this also change the JSON shape handled by the API? Does JSON somehow default to inline? Since this is an API model, a change to the JSON shape may break clients.

Reply:
@alexweav it defaults to inline, here is a simple example:

```
package main

import (
	"encoding/json"
	"fmt"
)

type Animal struct {
	Species string `json:"species"`
}

type Dog struct {
	Animal
	Breed string `json:"breed"`
}

func main() {
	var d Dog
	json.Unmarshal([]byte(`{"species": "canine", "breed": "labrador"}`), &d)
	fmt.Println(d)
}
```

```
go run main.go
{{canine} labrador}
```

Reply:
Makes sense. Thanks for the example @grobinson-grafana

Reply:
Out of convenience, I added both, even if the problem only exists when using the YAML parser. As George mentioned, it won't change the shape as it's just restating the default behavoir.